### PR TITLE
Updated response section

### PR DESCRIPTION
**Pull request by Nyholm**
Explain how you get domain objects, arrays or PSR7 responses

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,62 +80,36 @@ $mg->get("$domain/log", array('limit' => 25,
...
-        string(4) "info"
-        ["message_id"]=>
-        string(46) "20130820202406.24739.21973@samples.mailgun.org"
-      }
-    }
-  }
+$mg = new Mailgun("key-example", null, null, new ArrayDeserializer());
+$data = $mq->domains()->show('example.com');
+
+foreach ($data['receiving_dns_records' as $record) {
```
Comment (pirogoeth) [inline README.md]: `$data['receiving_dns_records']`

I think you missed a bracket :)

File: README.md [hunk 2]
```diff
@@ -80,62 +80,36 @@ $mg->get("$domain/log", array('limit' => 25,
...
 ```php
 $mg = new Mailgun("key-example");
-$domain = "example.com";
-
-$result = $mg->get("$domain/log", array('limit' => 25, 
-                                        'skip'  => 0));
-
-$httpResponseCode = $result->http_response_code;
-$httpResponseBody = $result->http_response_body;
+$dns = $mq->domains()->show('example.com')->getInboundDNSRecords();
```
Comment (pirogoeth) [inline README.md]: `$mq` -> `$mg`

**Comment by Nyholm** [comment]
Thanks for the reviews. The PR is updated

**Comment by Nyholm** [comment]
Can you try to repair the style CI key? Also, I do not think we need styleCI push to be required.

**Comment by pirogoeth** [comment]
Yeah, I tried -- hopefully it will work now.

**Comment by Nyholm** [comment]
Thank you

**Pull request merged**